src/modules/TelegramBot.js

```diff
@@ -495,4 +526,24 @@ function checkRoomFreeAtTimeslot(message, startDate, endDate, rooms) {
 			}
 		});
 }
+
+const informAdmins = (message) => {
+	const admins = loadUsers().find({ role: 'admin' });
+	admins.forEach((admin) => {
+		slimbot.sendMessage(admin.userId, message);
```

Comment:
this is just to send a notification for the admins to add the users in the backend? is there a way we could approve the users in telegram?

Reply:
hmm, this implementation doesn't allow admins to add in users manually, and imho, they shouldn't need to. if the users can self-register, it can save the admins manual work and prevent any need to restart the bot.

hmm, i think i'll update the main PR description with a summary of the registration process.